### PR TITLE
Change difficulty

### PR DESCRIPTION
**Pull request by SytzeAndr**
The game is more challenging. In my opinion, having a more difficult game is good to make the game more interesting to play. For the player it becomes more rewarding to win the game and to get a high score.
Ammo is spawned less, you can shoot less lasers, shark moves slower, enemies are bigger.
I also added the current amount of ammo on the screen. Quite useful for the player to know this information.

### CODE REVIEW

File: src/main/java/lasersharksgui/panes/GamePane.java [hunk 1]
```diff
@@ -113,8 +113,8 @@ public void resumeGame() {
   /**
    * Displays the score in the upper right corner of the screen.
    */
-  public void showScore() {
-    addText("Score: " + Highscores.getInstance().getScore(), TEXT_SCALE_SIZE_SMALL,
+  public void showScoreAndAmmo() {
```
Comment (Exclaminator) [inline src/main/java/lasersharksgui/panes/GamePane.java]: Can this method be made private?

**Comment by Exclaminator** [comment]
Still looks good.

File: src/test/java/lasersharks/seaobjects/AmmoTest.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void testGetWidthScale() {
    */
   @Test
   public void testGetAmount() {
-    final int amount = 10;
+    final int amount = 6;
```
Comment (spiderbiggen) [inline src/test/java/lasersharks/seaobjects/AmmoTest.java]: why does ammo give 6 ammo if the maximum ammo is 5 ?

Comment (SytzeAndr) [inline src/test/java/lasersharks/seaobjects/AmmoTest.java]: ¯\_(ツ)_/¯

Comment (SytzeAndr) [inline src/test/java/lasersharks/seaobjects/AmmoTest.java]: just kidding, actually I quite fast changed the test results to make it work without looking at it too much.

Comment (spiderbiggen) [inline src/test/java/lasersharks/seaobjects/AmmoTest.java]: it doesn't really matter either way.

I think this is ok to be merged then.

**Comment by spiderbiggen** [comment]
looks good

**Comment by MichielvdBerg** [comment]
I approve.

**Pull request merged**